dracut-saltboot/saltboot/module-setup.sh

```diff
@@ -29,8 +29,8 @@ get_python_pkg_deps_recursive() {
 
 # called by dracut
 installkernel() {
-    # for raid support, the kernel module is needed unconditionally, even in hostonly mode
-    hostonly='' instmods raid1
+    # for raid and crypt support, the kernel module is needed unconditionally, even in hostonly mode
+    hostonly='' instmods raid1 dm_crypt =crypto
```

Comment:
I missed this before, but in saltboot form for raid level we allow for 0,1,4,5,6 and 10. Shouldn't modules `raid0`, `raid10` and `raid456` be also installed? Or should saltboot formula be limited instead?

Reply:
We test only raid1. Anything else does not make sense in retail use case, IMO.
So if saltboot formula is considered retail-only, then I'd fix it there.

Reply:
Ok, let's fix sumaform form then.